BlockSettleUILib/AuthAddressDialog.cpp

```diff
@@ -406,10 +406,13 @@ void AuthAddressDialog::updateEnabledStates()
    if (selectionModel->hasSelection()) {
       const auto address = model_->getAddress(selectionModel->selectedRows()[0]);
 
+      auto tradeSettings_ = authAddressManager_->tradeSettings();
+      const bool allowSubmit = authAddressManager_->GetSubmittedAddressList().size() < tradeSettings_->authSubmitAddressLimit;
```

Comment:
Looks like that it will not work. GetSubmittedAddressList return funded address as well. So if there is already funded address - you will not be able to submit new.

Reply:
True, added flag to exclude submitted address. Thanks